src/styles/utils/variables.scss

```diff
@@ -0,0 +1,12 @@
+$card-width: 200px;
+$card-height: 408px;
+$prod-img-source: '../images/imac.jpeg';
+$prod-img-size: 160px, 134px;
+$font-size-default: 12px;
+$button-width: 166px;
+$button-height: 40px;
+$button-color: #00acdc;
+$button-font-color: #fff;
+$button-color--hover: #fff;
+$button-font-color--hover: #00acdc;
+$button-border-color--hover: solid, 1px, #00acdc;
```

Comment:
The value for `$button-border-color--hover` seems incorrect. According to CSS syntax, border properties should be defined separately or as a shorthand in a single string, like `1px solid #00acdc`. Consider revising this variable to match the correct CSS border property format.